assets/app/views/deployments.html

```diff
@@ -10,14 +10,112 @@
+                <span ng-switch="trigger.type">
+                  <span ng-switch-when="ImageChange">
+                    TODO: ImageChange                    
+                  </span>
+                  <span ng-switch-when="ConfigChange">                    
+                    TODO: ConfigChange                    
+                  </span>
+                  <span ng-switch-default>{{trigger.type}}</span>
+                </span>
+              </div>
+            </dl>
+
+            <dl class="dl-horizontal left indent">
+              <dt>Manual:</dt>
+              <dd>
+                <span>
+                  <button class="btn btn-primary" ng-click="" ng-disabled="">Start Deployment</button>
+                <span>
+              </dd>
+            </dl>
+
+            <dl class="dl-horizontal left indent">
+              <dt>Manual (CLI):
+                <a href="{{'start-deployment' | helpLink}}" target="_blank">
+                  <span class="learn-more-block">Learn more <i class="fa fa-external-link"> </i></span>
+                </a>
+              </dt>
+              <dd>
+                <code>osc deploy {{deploymentConfigName}} -n {{project.metadata.name}}</code>
+                <copy-to-clipboard-button clipboard-text="'osc deploy ' + deploymentConfigName + ' -n ' + project.metadata.name"></copy-to-clipboard-button>
+              </dd>
+
+            </dl>
+          </div>
+          
+          <div class="well" style="margin-bottom: 10px;" ng-repeat="deployment in deploymentsByDeploymentConfig[deploymentConfigName] | orderObjectsByDate : true">
+            
+            <h3>
+              Deployment #{{deployment.metadata.annotations.deploymentVersion}}
+              <span ng-if="deployment.metadata.annotations.deploymentVersion == deploymentConfig.latestVersion">(latest)</span>
+            </h3>
+            <div>Name: {{deployment.metadata.name}}</div>
+            <div>Created: <relative-timestamp timestamp="deployment.metadata.creationTimestamp"></relative-timestamp></div>            
+            <div>
+              <span>Status: {{deployment.metadata.annotations.deploymentStatus}}</span>
+              <span ng-switch="deployment.metadata.annotations.deploymentStatus" class="hide-ng-leave">
+                <span ng-switch-when="Running"> for <duration-until-now timestamp="deployment.metadata.creationTimestamp"></duration-until-now></span>
+                <span ng-switch-when="New"> for <duration-until-now timestamp="deployment.metadata.creationTimestamp"></duration-until-now></span>
+                <span ng-switch-when="Pending"> for <duration-until-now timestamp="deployment.metadata.creationTimestamp"></duration-until-now></span>
+              </span>
+              <span ng-switch="deployment.metadata.annotations.deploymentStatus" class="hide-ng-leave">
+                <span ng-switch-when="Complete" class="fa fa-check text-success" aria-hidden="true"></span>
```

Comment:
In terms of the "active" deployment, that would be the the latest "Complete" deployment. "Running" in terms of the persisted deployment status indicates the deployment *process* status. I hope that helps figure out how to distinguish...